Networking/Networking/Network/AlamofireNetwork.swift

```diff
@@ -96,8 +100,12 @@ public class AlamofireNetwork: Network {
                 .request(request)
                 .validateIfRestRequest(for: request)
                 .responseData { response in
-                    let result = response.result.toSwiftResult()
-                    promise(.success(result))
+                    if let error = response.networkingError {
+                        promise(.success(.failure(error)))
+                    } else {
+                        let result = response.result.toSwiftResult()
+                        promise(.success(result))
```

Comment:
super nit: just to be consistent with the other `responseData` mapping for easier code search
```suggestion
                        promise(.success(response.result.toSwiftResult()))
```

Reply:
Done in https://github.com/woocommerce/woocommerce-ios/pull/11110/commits/c2b39b943cea926d886ded70e10433f937b35d91